app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError("Error")
```

Comment:
The error message in the `ValueError` is not descriptive. Consider providing a more informative error message, such as 'Invalid command format'.